src/cmd/main.cpp

```diff
@@ -22,6 +22,7 @@
 #include <arpa/inet.h>
```

Comment:
is this our code? if no, we may have to credit or check the licence
if yes, add our copyright comments

Reply:
It has copyright information already.

Reply:
ok